client/src/components/Sort/sortType.js

```diff
@@ -1,8 +1,8 @@
-import keyMirror from 'keymirror';
+import keyMirror from "keymirror";
```

Comment:
_trivial_

Explicitly setting filter type here would appear beneficial for code in `SortOptionsList.jsx `

```javascript
// @flow

type SortType = {
  releaseDate: string,
  rating: string,
};

const SORT_TYPE: SortType = keyMirror({
  releaseDate: null,
  rating: null,
});
```